### DISCUSSION TITLE
Offline Features

### DISCUSSION THREAD

**Discussion by brianpetro** [brianpetro/smart-connect#3] [Polls]
Currently, all actions require Smart Connect to be actively connected. These are "online" features.

What if Smart Connect still worked when disconnected? These would be "offline" features.

**Comment by ottantaventi** [comment]
Question: I can use the customGPT ONLY if the Smart Connect app is running on a computer and is connected, right ?

Once this one app is running and connected, I can do my queries from whatever I want: another PC, my phone...
Correct ?

Is it possible to install the app also on a second PC ?
probably I've make sure that only ONE version at a time is running, correct ?

Anyway, traveling often, I would absolutely love, if the customGPT could be used, without the need to have actually the app running and connected on a pc.

**Reply by brianpetro** [reply]
> Question: I can use the customGPT ONLY if the Smart Connect app is running on a computer and is connected, right ?

Correct

> Once this one app is running and connected, I can do my queries from whatever I want: another PC, my phone... Correct ?

Correct

> Is it possible to install the app also on a second PC ? probably I've make sure that only ONE version at a time is running, correct ?

Yes, the app may be installed on multiple PCs, however, only one can be _connected_ at a time. If multiple instances try to connect, it will likely cause issues and downtime. This UX is something I hope to improve in the future.

> Anyway, traveling often, I would absolutely love, if the customGPT could be used, without the need to have actually the app running and connected on a pc.

Good to know. I expect the offline features will start with a subset of actions. For example, creating a note (or change a note) that would be applied to the vault next time Smart Connect is connected. But, depending on the response from the community, this anticipated direction may change.

Thanks for the feedback 😊
🌴
